src/main/java/com/spotinst/sdkjava/model/SpotOceanEcsClusterClient.java

```diff
@@ -143,4 +155,105 @@ public ClusterLaunchSpecification getLaunchSpec(OceanEcsClusterLaunchSpecRequest
 
         return retVal;
     }
+
+    public  OceanEcsCluster createOceanEcsCluster(OceanEcsClusterRequest oceanEcsClusterCreationRequest)
```

Comment:
Provide line breaks before and after if conditions.

Reply:
Incorporated.